shared/source/os_interface/linux/xe/ioctl_helper_xe.cpp

```diff
@@ -504,9 +511,10 @@ bool IoctlHelperXe::getTopologyDataAndMap(const HardwareInfo &hwInfo, DrmQueryTo
         topologySize -= itemSize;
         dataPtr = ptrOffset(dataPtr, itemSize);
     }
-
+    receivedEuPerDssInfo |= receivedSimd16EuPerDssInfo;
```

Comment:
why don't you simply add DRM_XE_TOPO_SIMD16_EU_PER_DSS to operate in the same var as DRM_XE_TOPO_SIMD16_EU_PER_DSS? you are not doing anything differently and kernel isn't going to report both. If the kernel reports both in future it actually means it has 2 different types of EUs in that platform, which would be very odd.